packages/server/src/api/muster/muster.router.ts

```diff
@@ -20,11 +19,12 @@ router.get(
   controller.getNearestMusterWindow,
 );
 
+const musterController = process.env.MUSTER_POSTGRES_CONTROLLER !== 'true' ? controller.getMusterRoster : musterPostgresController.getUserMusterCompliance;
```

Comment:
We really should remove this env variable this I feel like, see my other comments below.

Reply:
See my other comment.